lib/jekyll-compose/file_mover.rb

```diff
@@ -43,6 +44,20 @@ def move_file
         Jekyll.logger.info "#{resource_type_from.capitalize} #{from} was moved to #{to}"
       end
 
+      def update_front_matter
+        content = File.read(from)
+        if content =~ %r!\A(---\s*\n.*?\n?)^((---|\.\.\.)\s*$\n?)!m
+          content = $POSTMATCH
+          match = Regexp.last_match[1] if Regexp.last_match
+          data = movement.front_matter(Psych.safe_load(match))
+          File.write(from, "#{Psych.dump(data)}---\n#{content}")
+        end
+      rescue SyntaxError => e
```

Comment:
We can rescue just what needs to be rescued here: `Psych::SyntaxError`:
```suggestion
      rescue Psych::SyntaxError => e
```

Reply:
I agree.